terraform/modules/container-definition/main.tf

```diff
@@ -9,6 +9,9 @@ output "json_format" {
       command = ["/bin/bash", "-c", var.health_check]
     }
     image = var.image
+    linuxParameters = {
+      initProcessEnabled = true
```

Comment:
Interesting. This might also help in case any of our containers happen to be less than well-behaved in the zombie department 🧟 Apparently it uses https://github.com/krallin/tini

Reply:
Awesome, I didn't know about Tini, thanks!